Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -0,0 +1,31 @@
+﻿l_english: 
+ BTWK_story_event.0001.t:0 "Uther's Sickness"
+ BTWK_story_event.0001.desc:0 "You, Uther Pendragon, were once a mighty and well respected king in years past. However, thanks to a bout of illness, you have been reduced to shadow of your former self. Managing the realm, something that you would otherwise excel at, has become an impossible task in your newfound fraility. It's not uncommon you find yourself bedridden for days at a time coping with a high fever. \n You can't shake the feeling that you are not long for this world."
```

Comment:
Maybe mention something about the precarious state of succession in this event or the next one?